test/test_models.py

```diff
@@ -841,7 +841,7 @@ def test_video_model(model_fn, dev):
     # RNG always on CPU, to ensure x in cuda tests is bitwise identical to x in cpu tests
     x = torch.rand(input_shape).to(device=dev)
     out = model(x)
-    _assert_expected(out.cpu(), model_name, prec=0.1)
+    _assert_expected(out.cpu(), model_name, prec=1e-5)
```

Comment:
FYI @datumbox I am suspecting that this changing might cause flakyness in our internal tests

D38824237 shows that pytorch_vision_gpu-buck_2 is failing: https://www.internalfb.com/intern/testinfra/diagnostics/844425187765605.562950021904293.1660819094/

I don't see that on the other diffs (yet?) so I assume it's flaky.

Reply:
Thanks for the heads up. I would recommend turning off the internal test and relying on Github for verifying the models. Historically the `test_models.py` at Meta has been a source of flakyness, so feel free to turn off anything that breaks.

Reply:
Looking into it, could be that the expected files are wrong.